nexus/src/app/project.rs

```diff
@@ -79,6 +79,12 @@ impl super::Nexus {
             .lookup_for(authz::Action::CreateChild)
             .await?;
 
+        // TODO: We probably want to have "project creation", "resource
+        // provisioning creation", and "default VPC creation" co-located within
+        // a saga for atomicity.
+        //
+        // Until then, we just perform the operations sequentially.
```

Comment:
This will be accomplished within https://github.com/oxidecomputer/omicron/pull/2081